packages/api/src/augment/tx.ts

```diff
@@ -1622,4 +1623,9 @@ declare module '@polkadot/api/types/submittable' {
       vestOther: AugmentedSubmittable<(target: LookupSource | Address | AccountId | AccountIndex | string | Uint8Array) => SubmittableExtrinsic<ApiType>>;
     };
   }
+
+  export interface SubmittableExtrinsics<ApiType extends ApiTypes> extends AugmentedSubmittables<ApiType> {
+    (extrinsic: Uint8Array | string): SubmittableExtrinsic<ApiType>;
```

Comment:
So even in strict mode, this line is useful - it allows you to create an extrinsic directly from source. (Ad it actually does get used by some teams, it is a relatively late addition)

So would probably move this back. 

As for the next line, this we can augment in here on top of `SubmittableExtrinsics`

Or the other way around - always populate this interface, but in non-strict mode add the line below as well.

Reply:
Ok, was wrong about this - exactly as I explained - so all 100%.